packages/vitest/src/node/test-run.ts

```diff
@@ -57,7 +57,7 @@ export class TestRun {
     }
   }
 
-  async end(specifications: TestSpecification[], errors: unknown[], coverage?: unknown): Promise<void> {
+  async end(specifications: TestSpecification[], errors: unknown[], meta: { coverage?: unknown; executionTimes?: number[] }): Promise<void> {
```

Comment:
why would times be optional?

Reply:
Only present when `--merge-reports` is used.